server/src/main/java/com/cloud/template/TemplateManagerImpl.java

```diff
@@ -300,6 +300,9 @@
     @Inject
     private EndPointSelector selector;
 
+    private final static String VMWARE_TOOLS_ISO = "vmware-tools.iso";
```

Comment:
Just an improvement suggestion: I've noticed these ISO names are hardcoded on the Vmware and Xen plugins codebase. What about moving `VMWARE_TOOLS_ISO` and `XS_TOOLS_ISO` to the `TemplateManager` interface and replace the hardcoded ISO names for these values on the `plugins/hypervisor/vmware` and `plugins/hypervisor/xenserver` modules?

Reply:
@nvazquez done